package.json

```diff
@@ -29,12 +30,16 @@
     "@nebula.js/cli-sense": "1.7.0",
     "@nebula.js/cli-serve": "1.7.0",
     "@nebula.js/stardust": "1.7.0",
+    "@testing-library/jest-dom": "^5.16.4",
     "babel-eslint": "10.1.0",
-    "scriptappy-from-jsdoc": "^0.7.0",
+    "babel-jest": "^28.1.3",
```

Comment:
Please use the static version, '28.1.3'. and Others as well.

Reply:
ok thank you I will change